src/hpc/tls.c

```diff
@@ -78,39 +79,27 @@ Int AllocateExtraTLSSlot() {
 
 void InitTLS()
 {
-  void InitScannerTLS();
-  void InitStatTLS();
-  void InitExprTLS();
-  void InitCoderTLS();
-  void InitThreadAPITLS();
-  void InitOpersTLS();
-  void InitAObjectsTLS();
-  InitScannerTLS();
-  InitStatTLS();
-  InitExprTLS();
-  InitCoderTLS();
-  InitThreadAPITLS();
-  InitOpersTLS();
-  InitAObjectsTLS();
+  GAPState *state = GetTLS();
+  InitScannerState(state);
+  InitStatState(state);
+  InitExprState(state);
+  InitCoderState(state);
+  InitThreadAPIState(state);
+  InitOpersState(state);
+  InitAObjectsState(state);
```

Comment:
If `globalstate.c` was compiled for HPC-GAP, too, then perhaps we could / should call `InitGAPState` from here?